pkg/tasks/prometheusadapter.go

```diff
@@ -255,6 +231,14 @@ func (t *PrometheusAdapterTask) create(ctx context.Context) error {
 		}
 	}
 
+	{
+		// TODO: Remove this in 4.16
+		err := t.client.DeleteServiceMonitorByNamespaceAndName(ctx, t.client.Namespace(), "kubelet-resource-metrics")
```

Comment:
unfortunate copy/past :)
You meant to delete the `adapter-config-dedicated-sm` configmap I think
(I think Daniel has added some checks for unused fcts?)
For being too picky I'll create a ticket to get rid of this code in 4.16 :)

Reply:
oh dear...unfortunate? I'd rather go with vim fat fingering on my part. Thanks for cleaning up after me :grimacing: .

Fixed now.
I'm also a little surprised that golang-ci didn't catch that.

Reply:
I'll ping him here if he has an idea (cc @danielmellado), otherwise I'll dig it later.